smoke/LeafDevice/LeafDevice.cs

```diff
@@ -27,6 +27,7 @@ public async Task RunAsync()
                 await GetOrCreateDeviceIdentity();
                 await ConnectToEdgeAndSendData();
                 await this.VerifyDataOnIoTHub();
+                await this.VerifyDirectMethod();
```

Comment:
Are we planning to add more checks here, like GetTwin, update reported properties, etc?

Reply:
Not right now. The possibilities are endless. For this PBI this is the last scenario, but I encourage Mike to keep an ongoing E2E Effort so we can add more coverage.